index.html

```diff
@@ -22,31 +22,34 @@
   <body>
     <div class="page-container">
       <div class="header1">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
-      </header>
-    </div>
-<div class="header2">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><ul class="navlist">
-          <li> <a href="#">Portfolio</a></li>
-          <li> <a href="#">About</a></li>
-          <li> <a href="#">Contact</a></li>
-        </ul></nav>
-      </header>
-    </div>
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav><i class="fa-solid fa-bars hambrgricon" id="hmbg"></i></nav>
+        </header>
+      </div>
+      <div class="header2">
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav>
+            <ul class="navlist">
+              <li class="linkitems"><a href="#portfolio">Portfolio</a></li>
+              <li class="linkitems"><a href="#about">About</a></li>
+              <li class="linkitems"><a href="#contact">Contact</a></li>
+            </ul>
+            <i class="fa-solid fa-times closeicon" id="closebtn"></i>
+          </nav>
+        </header>
+      </div>
       <section class="content">
-        <h1 class="boldtext">I’m Megha <br>
-          Glad to see you!</h1>
+        <h1 class="boldtext">
+          I’m Megha <br />
+          Glad to see you!
+        </h1>
         <p class="intro">
-          I’m a software developer! I can help you 
-          build a product , feature or website Look
-          through some of my work and
-          experience! If you like what you see and
-          have a project you need coded, don’t
-          hestiate to contact me.
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
         </p>
         <div>
           <p class="socialmediatag">LET'S CONNECT</p>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove the extra space on the right as it makes your project doesn't match the Figma template. Below is how it looks:

| Styling issues | Styling issues |
| --------------- | ---------------- |
| ![Screenshot from 2022-07-28 13-42-07](https://user-images.githubusercontent.com/83042742/181497394-28dccbfb-64a9-4c70-bf55-1aff073fbc41.png) | ![Screenshot from 2022-07-28 13-46-45](https://user-images.githubusercontent.com/83042742/181497594-3b298884-d01a-467f-a817-88d0cc9e72fb.png) |